### PR TITLE
DOCSP-29359: Update legacy version flipper for 7.0 upcoming

### PR DESCRIPTION
**Pull request by mdb-ashley**
We're publishing a new version of the docs, v7.0. v7.0 content lives on the master branch, and v6.3 content lives on the v6.3 branch.

### CODE REVIEW

**Review by mmeigs — APPROVED** [review]
LGTM!

File: data/manual-published-branches.yaml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-    - '6.1 (Rapid Release)'
+    - '7.0 (Upcoming)'
+    - '6.3 (Rapid Release)'
     - '6.0 (Major Release)'
     - '5.0'
     - '4.4'
     - '4.2'
   active:
-    - '6.1'
+    - '6.3'
```
Comment (rayangler) [inline data/manual-published-branches.yaml:10]: Should probably add 7.0 as an active version as well.

```suggestion
    - '7.0'
    - '6.3'
```

**Review by rayangler — COMMENTED** [review]
Just wanted to bring up a couple of things that may be missing based on a [previous/similar commit](https://github.com/mongodb/docs-tools/commit/0be6a49f38ee4da9840939e7040f3f1183a3d05c). Feel free to ignore if intention is different or if unnecessary!

File: data/manual-published-branches.yaml [hunk 2]
```diff
@@ -1,18 +1,19 @@
 version:
   published:
-    - '6.1 (Rapid Release)'
+    - '7.0 (Upcoming)'
+    - '6.3 (Rapid Release)'
```
[the remark this answers is not in the document]
Comment (mdb-ashley) [reply data/manual-published-branches.yaml:4]: Done!

File: data/manual-published-branches.yaml (hunk 1 above)
Comment (mdb-ashley) [reply data/manual-published-branches.yaml:10]: Done!

**Review by rayangler — APPROVED** [review]

**Pull request merged**